scribus/plugins/import/pdf/slaoutput.cpp

```diff
@@ -3042,11 +3047,13 @@ void SlaOutputDev::updateFont(GfxState *state)
 	tmpBuf = nullptr;
 	fontLoc = nullptr;
 
-	if (!(gfxFont = state->getFont())) {
+	if (!(gfxFont = state->getFont()))

```

Comment:
imo you can submit in a separate patch  the "code layout" that are clearly separated from what you are programming.

if they have no change in the behavior they will be accepted as is and your patch will be smaller and jean can focus on the important things : - )